packages/alert/src/lib/alert.tsx

```diff
@@ -0,0 +1,79 @@
+import MUIAlert, {
+  AlertProps as MUIAlertProps,
+  AlertColor,
+} from '@mui/material/Alert';
+import { styled } from '@novatics/styles';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import CheckCircleIcon from '@mui/icons-material/CheckCircleOutline';
+import Box from '@mui/material/Box';
+import { Typography } from '@mui/material';
+
+/* eslint-disable-next-line */
+export interface AlertProps extends MUIAlertProps {
+  severity?: AlertColor;
+  onClose?: () => void;
+  title?: string | undefined;
+  action?: React.ReactNode;
+}
+
+const CloseButton = styled(IconButton)`
+  padding: 4px;
+  border-radius: 5px;
+  color: black;
+`;
+
+const Content = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  padding-top: 2px;
+`;
+
+export const Alert = (props: AlertProps) => {
+  const {
+    children,
+    title,
+    severity = 'info',
+    variant = 'outlined',
+    onClose,
+    action,
+    ...other
+  } = props;
+
+  const severityIcons = {
+    success: <CheckCircleIcon fontSize="small" />,
+  };
+
+  return (
+    <MUIAlert
+      variant={variant}
+      severity={severity}
+      icon={severityIcons[severity]}
+      action={
+        <Box sx={{display: 'flex', alignItems: 'center'}}>
+          {action ? <Box mr="4px">{action}</Box> : null}
```

Comment:
Action don't repect the distance on figma

![image](https://github.com/Novatics/novatics-ui/assets/15728617/6ceccdd3-2d16-4e68-9f70-c034aac9eb21)
![image](https://github.com/Novatics/novatics-ui/assets/15728617/8a556b6d-03d1-4d6e-983d-439181f27a07)